File: Assignments/Assignment 26/C Code/program26_1.c
// Input : 5 
// Output : A   B   C   D   E

#include<stdio.h>

void Pattern(int iNo)
{
    int iCnt = 0;
    int iAlphabet = 65;     // ASCII OF 'A' IS 65
    
    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        printf("%c\t", iAlphabet);
        iAlphabet++;
    }
}

int main()
{
    int iValue = 0;

    printf("Enter number of elements : \n");
    scanf("%d", &iValue);

    Pattern(iValue);

    return 0;
}

File: Assignments/Assignment 26/Java Code/program26_1.java
// Input : 5 
// Output : A   B   C   D   E

import java.util.*;

class Pattern {
    void Display(int iNo) {
        int iCnt = 0;
        char iAlphabet = 65; // ASCII OF 'A' IS 65

        for (iCnt = 1; iCnt <= iNo; iCnt++) {
            System.out.print(iAlphabet + "\t");
            iAlphabet++;
        }
    }
}

class program26_1 {
    public static void main(String A[]) {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0;

        System.out.println("Enter number of elements : ");
        iValue = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue);
    }
}